tsconfig.json

```diff
@@ -23,7 +23,7 @@
     "esModuleInterop": true,
     "experimentalDecorators": true,
     "useDefineForClassFields": false, /* See https://lit.dev/docs/components/properties/#avoiding-issues-with-class-fields */
-    "removeComments": true,
+    "removeComments": false,
```

Comment:
Not sure why this was set to `true` in the first place, but I'm curious to know what it was affecting.